### DISCUSSION TITLE
Перестала работать функция linkToPanel если указан параметр fields

### DISCUSSION THREAD

**Discussion by gorbarov** [totumonline/totum-ru-issues-and-discussions#399] [ОШБК!] [answered]
Проблема такая, в какой-то момент перестал работать во всех таблицах код действия вида:

```
= : linkToPanel(table: $#ntn; id: #id; fields: $fields;  refresh: true; columns: 1;  field: 'data_plan' = '';)
	fields: listCreate(item: "data_plan"; item: "reason_why")
```

Открывается пустое окошко:

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/388620/66d276d8-354b-468f-b39f-2d3de7a1b9d4)

Убираешь параметр fields и всё работает. При том, что эти таблицы никто не трогал. Всё работало, а в какой-то момент просто перестало, причём во всех таблицах, где вызывается linkToPanel с параметром fields.

Куда копать, чё смотреть? Версия на скрине.
Сильно не хочется отказываться от такого удобного параметра.

**Comment by ttm-alexeypolunin** [comment]
Ошибка в консоли есть?

**Reply by gorbarov** [reply]
Ага, 
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/388620/f2ca2fbf-60d4-41e3-ae66-8dece2599265)

**Reply by ttm-alexeypolunin** [reply, marked as answer]
Мы что-то там поправили, но не факт, что оно чинит эту ошибку тк источник ошибки до конца не ясен.

**Reply by gorbarov** [reply]
... и всё заработало как было )))
